docs/guides/latest-updates.mdx

```diff
@@ -29,6 +29,17 @@ Jump to [Qiskit SDK](#latest-qiskit-sdk) | [Qiskit Runtime client](#latest-qiski
 
 This section summarizes the recent enhancements and new features for IBM Quantum Platform.
 
+**8 May 2025**
+
+- [Flex Plan](https://www.ibm.com/quantum/blog/flex-plan), a new resource consumption offering, has been released.
+- [Tutorials](https://quantum.cloud.ibm.com/docs/tutorials) are now available on the new platform to help users with use-case exploration.
```

Comment:
Given that this is inside `<CloudContent>`, it should be safe to make this change:

```suggestion
- [Tutorials](/docs/tutorials) are now available on the new platform to help users with use-case exploration.
```